labelbox/schema/ontology.py

```diff
@@ -124,3 +216,45 @@ def convert_keys(json_dict: Dict[str, Any],
     if isinstance(json_dict, list):
         return [convert_keys(ele, converter) for ele in json_dict]
     return json_dict
+
+
+@dataclass
+class OntologyBuilder:
```

Comment:
Add a docstring to explain what this is and how it should be used. This is super confusing that an ontology is independent of a project yet you can only create one when you setup a project. This is a limitation of the backend but lets just give a little explanation for what this is and how it should be used.